src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -62,6 +63,8 @@ namespace QgsWfs
       const QString &geometryName;
 
       const QgsCoordinateReferenceSystem &outputCrs;
+
+      bool forceGeomToMulti;
```

Comment:
Initialize?

Reply:
If I initialize `forcegeomToMulti` to `false`, the build failed with:
```
...
../src/server/services/wfs/qgswfsgetfeature.cpp:65:43: warning: non-static reference ‘const QgsCoordinateReferenceSystem& QgsWfs::{anonymous}::createFeatureParams::outputCrs’ in class without a constructor [-Wuninitialized]
       const QgsCoordinateReferenceSystem &outputCrs;
                                           ^~~~~~~~~
../src/server/services/wfs/qgswfsgetfeature.cpp: In function ‘void QgsWfs::writeGetFeature(QgsServerInterface*, const QgsProject*, const QString&, const QgsServerRequest&, QgsServerResponse&)’:
../src/server/services/wfs/qgswfsgetfeature.cpp:413:41: error: could not convert ‘{layerPrecision, layerCrs, attrIndexes, typeName, withGeom, geometryName, outputCrs, forceGeomToMulti}’ from ‘<brace-enclosed initializer list>’ to ‘const QgsWfs::{anonymous}::createFeatureParams’
```